.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -58,6 +58,9 @@ assignees: ''
       Suggested approach: Follow the full [GKE getting started guide].
 - [ ] Check draft release from [releases] page and publish the release
 - [ ] Announce the release in #general on Slack (only [@]channel for vX.Y.0)
+- [ ] Update Grafana dashboards (only for vX.Y.0)
+  - Install the dashboards available in ``./examples/kubernetes/addons/prometheus``
+    and use them to upload them to Grafana.com.
```

Comment:
Does "Install" mean follow the directions in the README.md in that directory to create prometheus+grafana in your cluster? And then there's an upload button from the UI?

Reply:
Seems like somewhere in the overall development process we should also be doing a `make -C examples/kubernetes/addons/prometheus`, since that's currently showing a diff for me:

```
diff --git a/examples/kubernetes/addons/prometheus/monitoring-example.yaml b/examples/kubernetes/addons/prometheus/monitoring-example.yaml
index 7e5541543a78..fd33286bbfd7 100644
--- a/examples/kubernetes/addons/prometheus/monitoring-example.yaml
+++ b/examples/kubernetes/addons/prometheus/monitoring-example.yaml
@@ -4475,7 +4475,7 @@ data:
               "expr": "avg(rate(cilium_nodes_all_events_received_total{k8s_app=\"cilium\"}[1m])) by (pod, event_type, source) * 60",
               "format": "time_series",
               "intervalFactor": 1,
-              "legendFormat": "{{event_type}} {{source}}",
+              "legendFormat": "{{eventType}} {{source}}",
               "refId": "B"
             }
           ],
```

Reply:
>Does "Install" mean follow the directions in the README.md in that directory to create prometheus+grafana in your cluster? And then there's an upload button from the UI?

@joestringer, no, there's an "share" button that exports as a json file for which we need to upload it to grafana.com manually.

Reply:
Is that the same as `examples/kubernetes/addons/prometheus/files/grafana-dashboards/*.json`, or different?

Reply:
It's different because it will create some fields that can be variable, i.e. the prometheus URL, where in our jsons they seem to be statically configured to use the prometheus instance that is created in the `monitoring-example.yaml`

Reply:
OK thanks for clarifying.

I think that the comment as-is is enough to signal that something needs to be done, but as evidenced by my line of questioning above I'm not sure if it's obvious enough how to actually do it. It's a pity it requires some manual UI interaction to get the proper format out :(

That said, we can probably figure it out when it comes down to the actual release. No doubt we'll go through this step during v1.10 and we can always revise whether there's more detail we need to add to the description here.